### PR TITLE
change log for 2.5.0

### PR DESCRIPTION
**Pull request by banisadr**
### Description

Change log for 2.5.0 release. Note: calling out NPM package rename in highlights section at the top because its important for people to be aware of.

### CODE REVIEW

File: changelog/v2.5.0.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+date: "2024-05-24T04:12:48"
+hide_table_of_contents: true
+description: "Change log for release 2.5.0 of Foxglove app"
```
Comment (jtbandes) [inline changelog/v2.5.0.mdx:4]: Pretty underwhelming description. Looks like 2.4 and 2.3 had no description. 2.2 had "🌎 Logarithmic depth, distraction-free service calls, improved JSON support, and fewer errors." which is kinda nice. I think we should either go with more details or no description.

File: changelog/v2.5.0.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## ⏱️ DateTime and 24-hour time formats
+
+Visualization playback now supports displaying timestamps in RFC 3339 (ISO 8601 DateTime) and 24-hour formats in addition to 12-hour and epoch.
+
+![DateTime Formats GIF](/img/changelog/2.5.0/time-formats.webp)
+
+## ✍️ Renamed NPM package
+
+When we [launched Foxglove 2.0](https://foxglove.dev/blog/foxglove-2-0-unifying-robotics-observability#from-visualization-to-observability), we dropped the "Studio" name. In keeping with that, we renamed the `@foxglove/studio` NPM package to `@foxglove/extension`. Please update your custom extensions to depend on `@foxglove/extension` to receive future SDK updates.
```
Comment (jtbandes) [inline changelog/v2.5.0.mdx:21]: Intentional to link to a specific section of the blog post vs the post as a whole?

File: changelog/v2.5.0.mdx (hunk 1 above)
Comment (banisadr) [reply changelog/v2.5.0.mdx:4]: Updated!

File: changelog/v2.5.0.mdx (hunk 2 above)
Comment (banisadr) [reply changelog/v2.5.0.mdx:21]: Yes - we talk about combining the app there and have a nice fun meme about dropping the "Data Platform" name.

**Review by jtbandes — APPROVED** [review]

**Pull request merged**